Doppler.Sap/Controllers/BillingController.cs

```diff
@@ -82,5 +82,11 @@ public async Task<IActionResult> CancelCreditNote([FromBody] CancelCreditNoteReq
 
             return new OkObjectResult("Successfully");
         }
+
+        [HttpGet("{billingSystemId}/Invoices/{dopplerInvoiceId}")]
+        public async Task<InvoiceResponse> CreateCreditNote([FromRoute] int billingSystemId, [FromRoute] int dopplerInvoiceId)
```

Comment:
I think that CreateCreditNote should be in a  `CreditNoteController.cs`

Reply:
I thought having only one controller for billing. The billing contains Invoices, payments and credit notes. But if for you more clear separate it i don't have any problem.

Reply:
Is it ok that this method call `CreateCreditNote` and not for example `GetInvoice`?

Reply:
Yes, the name is incorrect. The problem was when I resolved the conflict. Now was updated

Reply:
Done